corehq/apps/export/tests/test_export_conversion.py

```diff
@@ -332,6 +338,40 @@ def test_system_property_conversion(self):
             self.assertEqual(column.selected, selected, '{} selected is not {}'.format(path, selected))
 
 
+class TestRevertNewExports(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.new_exports = [
+            FormExportInstance(),
+            CaseExportInstance(),
+        ]
+        for export in cls.new_exports:
+            export.save()
+
+    @classmethod
+    def tearDownClass(cls):
+        for export in cls.new_exports:
+            export.delete()
```

Comment:
Looks like the test cases mutate the exports in `cls.new_exports`, should these be `setUp` and `tearDown` instead of `..Class`?